core/src/main/scala/kafka/security/authorizer/AclAuthorizer.scala

```diff
@@ -304,25 +302,26 @@ class AclAuthorizer extends Authorizer with Logging {
   }
 
   private def matchingAcls(resourceType: ResourceType, resourceName: String): Set[Acl] = {
-    inReadLock(lock) {
```

Comment:
We can change the lock to be a regular lock instead of read/write lock now since we don't use the read lock any more?

Reply:
Good point.

Reply:
I've updated the code.